### PR TITLE
Adjust Role Titles to Use Admin and Editor

### LINKED ISSUE #98: Adjust Role Titles
The original roles (Researcher, Volunteer, and Viewer) should change to better comply with existing Panoptes roles. The new roles will be:

Admin - full access
Editor - full access sans approval
Viewer - minimal access

### PR DESCRIPTION
**Pull request by wgranger**
Closes #98 

Mainly just a change in terminology. This is to avoid the former confusion where a user with a PFE role of "researcher" was not given the "researcher" role in ALICE.

### CODE REVIEW

**Comment by wgranger** [comment]
I think I need to optimize it a bit to close #84. If you are associated with more than 50 projects, it might be that your role for one of the text editor projects isn't being retrieved.

I'm going to try to get a collection of project roles based on projects in the text editor instead of _all_ projects associated with a user.

**Comment by wgranger** [comment]
Closes #84. Alright, give this another go. I think this should fix the issue, but let me know if not. Either way, this closes another issue linked.

File: src/store/ProjectsStore.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ import apiClient from 'panoptes-client/lib/api-client.js'
...
-  VOLUNTEER: 'Volunteer',
+  ADMIN: 'Admin',
+  EDITOR: 'Editor',
   VIEWER: 'Viewer'
 }
 
-const researcherRoles = ['owner', 'collaborator']
-const volunteerRoles = ['expert', 'researcher', 'moderator']
+const adminRoles = ['owner', 'collaborator']
+const editorRoles = ['expert', 'researcher', 'moderator']
```
Comment (mcbouslog) [inline src/store/ProjectsStore.js]: Oh I think I see why it's still happening! I think `'scientist'` needs to be added to this array, per 
Screen Shot 2020-02-26 at 10 35 16 AM
Oy the roles are a quagmire! But I think if this should do it.

Comment (wgranger) [reply src/store/ProjectsStore.js]: Panoptes defines the "researcher" role as a "scientist?" That's very confusing. I'll make the edit and ask you to give it one more spin to make sure all is sorted out. Thanks for your tedious review.

**Comment by wgranger** [comment]
Good eye on the review. This should be good to go.

**Pull request merged**